playbooks/common/openshift-cluster/upgrades/etcd/rhel_tasks.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Verify cluster is healthy pre-upgrade
+  command: "etcdctl --cert-file /etc/etcd/peer.crt --key-file /etc/etcd/peer.key --ca-file /etc/etcd/ca.crt -C https://{{ openshift.common.hostname }}:2379 cluster-health"
```

Comment:
I don't think the cert path will change and I'd someone to specify --data-dir as necessary.